tests/textcase/lsp/fixtures/camel-case.ts

```diff
@@ -0,0 +1,5 @@
+export const doSomething = () => {
```

Comment:
I thought it's not really necessary to have a React component. Makes it easier to work with.

Reply:
Mhh good point. I checked and I can't feel that much of a difference. So I could probably just change it back.